core/translate/planner.rs

```diff
@@ -494,6 +484,49 @@ pub fn prepare_select_plan<'a>(schema: &Schema, select: ast::Select) -> Result<P
     }
 }
 
+pub fn prepare_delete_plan(
+    schema: &Schema,
+    tbl_name: &QualifiedName,
+    where_clause: Option<Expr>,
+) -> Result<Plan> {
+    let table_name = tbl_name.name.0.clone();
```

Comment:
you need to use `normalize_ident()` here to make sure equivalent expressions like `users`, `"users"` and `[users]` all reference the same table correctly

Reply:
Applied!